app/src/main/java/org/simple/clinic/appupdate/AppUpdateConfig.kt

```diff
@@ -1,6 +1,17 @@
 package org.simple.clinic.appupdate
 
+import org.simple.clinic.remoteconfig.ConfigReader
+
 data class AppUpdateConfig(
     val inAppUpdateEnabled: Boolean,
     val differenceBetweenVersionsToNudge: Long
-)
+) {
+
+  companion object {
+    fun read(reader: ConfigReader): AppUpdateConfig {
```

Comment:
Reading from Firebase config involves disk I/O, so we should inject an `Observable<AppUpdatedConfig>` and use `Observable.fromCallable {}` to read the config instead.

See `PatientConfig#read(ConfigReader)`.

Reply:
Done!